plaid-stl/src/github/mod.rs

```diff
@@ -29,6 +29,30 @@ pub struct RepositoryCollaborator {
     pub permissions: Permission,
 }
 
+// Structs used for deserializing GH's responses when listing Copilot seats
+
+#[derive(Debug, Deserialize)]
+/// A person assigned to a seat in a Github Org Copilot subscription
+pub struct CopilotAsignee {
+    pub login: String,
+    #[serde(rename = "type")]
+    pub type_: String,
```

Comment:
What does this type represent?

Reply:
Maybe just some comments describing it

Reply:
Done. The schema doesn't seem too expressive.
```
              "type": {
                "type": "string",
                "examples": [
                  "User"
                ]
              },
```